bylaws.tex

```diff
@@ -299,7 +299,7 @@
 \bsection{Housing Priority System}
 The Housing Priority System is a means for determining the priority a House member has in a Housing issue such as Single Room Assignment or the Assignment of Available Housing.
 The member with top priority is the member with on-floor status and the most Housing Priority Points.
-Housing points are accumulated during the Housing Evaluation described in \ref{Housing Evaluation}.
+Housing points are accumulated by the end of the year in accordance with the Evaluations Director. \ref{Housing Points}.
```

Comment:
> accordance with the Evaluations Director
Doesn't feel right. I don't even think that's proper grammar.
I agree this should be updated, but I'm not sure that's the correct way to say it

Reply:
Let's yeet all of this and just make housing points attainable at the end of spring evals